.gitignore

```diff
@@ -53,7 +53,6 @@ node_modules/
 /config/master.key
 
 config/settings.local.yml
-config/settings/*.local.yml
```

Comment:
Please don't, this might mess up other people's settings. There are other keys in these files that might to be changed.